components/gitpod-db/src/periodic-deleter.ts

```diff
@@ -65,7 +65,7 @@ export class PeriodicDbDeleter {
         for (const i in rows) {
             const row = rows[i];
             const whereClause = whereClauseFn(row);
-            deletions.push(`DELETE FROM ${table.name} WHERE ${whereClause};`);
+            deletions.push(`DELETE FROM ${table.name} WHERE ${whereClause} AND ${deletionColumn} = true;`);
```

Comment:
For my understanding, this ensures that between the time when we run the select, and the time when we run the DELETE, the deletion field remains unaffected